src/js/components/DataTable/stories/Simple.js

```diff
@@ -10,6 +10,12 @@ const SimpleDataTable = () => (
   <Grommet theme={grommet}>
     <Box align="center" pad="large">
       <DataTable columns={columns} data={DATA} step={10} />
+      <Box margin={{ top: 'medium' }} align="center">
+        <Anchor
```

Comment:
Can you check if we can move the `margin={{ top: 'medium' }} align="center"` to the Anchor level? and then eliminate the Box? Anchor has `alignSelf`.

Reply:
Or maybe we don't really need the `align`, to keep it simple.

Reply:
changes made. I added align only because I think it looks nicer.